app/config.py

```diff
@@ -25,8 +25,6 @@ class QueueNames:
     SMS_CALLBACKS = "sms-callbacks"
     ANTIVIRUS = "antivirus-tasks"
     SANITISE_LETTERS = "sanitise-letter-tasks"
-    SAVE_API_EMAIL = "save-api-email-tasks"
-    SAVE_API_SMS = "save-api-sms-tasks"
```

Comment:
not sure if this code still in use but it seems there is mention of these Queues [here](https://github.com/alphagov/notifications-api/blob/d0f90b70a8ac83f00df581a484e5eea0dd309983/app/v2/notifications/post_notifications.py#L304)

Reply:
think this branch could be rebased but those lines of code were removed already here:

https://github.com/alphagov/notifications-api/pull/4124/files#diff-cc69ccd3b3b0ba4e44f73c182412190d6af4e52c1bc0680fdc745a60c2eced22L305

Reply:
i've now rebased 👍